main.py

```diff
@@ -84,10 +108,88 @@
+                        await client.get_channel(message.channel).send(msg)
+                        element.members.append(buddy)
+
+                intro_msg = creation_messages.cc_intro(element.members)
+
+                viewers = []
+                abductees = []
+                for member in db.player_list():
+                    if db_get(member,'abducted') == 1:
+                        abductees.append(member)
+                    else:
+                        if member in element.members or db_get(member,'role') in ['Dead','Spectator']:
+                            viewers.append(member)
+
+                # Role objects (based on ID)
+                main_guild = botspam_channel.guild # Find the guild we're in
+                roles = main_guild.roles # Roles from the guild
+                game_master_role = discord.utils.find(lambda r: r.id == game_master, roles)
+                dead_participant_role = discord.utils.find(lambda r: r.id == dead_participant, roles)
+                frozen_participant_role = discord.utils.find(lambda r: r.id == frozen_participant, roles)
+                default_permissions = {
+                    main_guild.default_role: discord.PermissionOverwrite(read_messages=False),
+                    frozen_participant_role: discord.PermissionOverwrite(send_messages=False),
+                    dead_participant_role: discord.PermissionOverwrite(read_messages=True, send_messages=False),
+                    game_master_role: discord.PermissionOverwrite(read_messages=True),
+                    client.user: discord.PermissionOverwrite(read_messages=True,send_messages=True),
+                    **{
+                        member: discord.PermissionOverwrite(read_messages=True) for member in viewers
+                    },
+                }
+
+                # Create a new category if needed
+                if db.get_category() == None:
+                    category = await main_guild.create_category('CC part {}'.format(db.count_categories(), reason='It seems like we couldn\'t use our previous category! Don\'t worry, I just created a new one.')
+                    db.add_category(category)
+                else:
+                    category = db.get_category()
+
+                try:
+                    # Create the text channel
+                    channel = await main_guild.create_text_channel(
+                        name="s{}_".format(config.season) + element.name,
+                        category=category,
+                        overwrites=default_permissions,
+                        reason='CC requested by ' + message.author.mention)
+                    db.add_channel(channel.id,element.owner)
+                    await channel.send(intro_msg)
+
+                    # Set all access rules in the database
+                    for victim in abductees:
+                        db.set_user_in_channel(channel.id,victim,3)
```

Comment:
I don't think this sets permissions on the channel itself, unless I'm missing something. Even if it did, it would spam the audit log.

Reply:
Correct. It doesn't.

The reason we're setting their values to three, (though effectively, their permissions are the same as any other non-invited participant,) is so we know we will have to add them back to this conspiracy channel when they get out of the swamp. We do want them here, but not to the wolves' conspiracy channel that wasn't AT ALL waiting for a peeking innocent to All-Star-door-slam their tea party.

Reply:
kk